deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -75,17 +75,16 @@ def collect_screenshots_for_languages(path)
 
         prefer_framed = Dir.glob(File.join(lng_folder, "*_framed.#{extensions}"), File::FNM_CASEFOLD).count > 0
 
+        UI.important("Framed screenshots are detected! 🖼 Non-framed screenshot files may be skipped. 🏃") if prefer_framed
```

Comment:
Shouldn't we only display this if the user actually has framed screenshots? I'm sure about 90% of the `deliver` users don't use `frameit` for their screenshots.